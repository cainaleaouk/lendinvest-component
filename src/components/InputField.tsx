import styled from 'styled-components';
import { Colors } from '../themes';

export const InputField = styled.input`
    border: solid 1px ${Colors.GREY};
    color: ${Colors.GREY_DARK};
    text-align: right;
    padding: 8px;
    ::-webkit-inner-spin-button{
        -webkit-appearance: none; 
        margin: 0; 
    }
    ::-webkit-outer-spin-button{
        -webkit-appearance: none; 
        margin: 0; 
    }    
`;