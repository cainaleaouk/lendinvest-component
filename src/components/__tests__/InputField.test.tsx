import React from 'react';
import renderer from 'react-test-renderer';
import { InputField } from '../InputField';

describe('<InputField /> snapshot', () => {
    it('should render correctly', () => {
        const component = renderer.create(
          <InputField />
        );
        
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
})

