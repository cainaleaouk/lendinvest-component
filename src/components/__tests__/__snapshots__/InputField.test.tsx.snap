// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputField /> snapshot should render correctly 1`] = `
<input
  className="sc-bdfBwQ iVJIjv"
/>
`;
